### PR TITLE
update nobel-prizes demo to v0.28

### PR DESCRIPTION
**Pull request by oluademola**
# Pull Request

## What does this PR do?
**Update Nobel-prizes Demo to v0.28**
- This update does not introduce any code-related breaking changes.
- Bumped meilisearch-js version to v0.27.0
- Bumped instant-meilisearch to 0.8.0
- Set `keepZeroFacets: true`

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by bidoubiwa** [comment]
There is a mis-use of package-lock.json and yarn.lock. 

You made the upgrade using `npm`, instead, could you make it using yarn.lock?

You can delete `package-lock.json` and just `yarn install` to update your yarn.lock with the changes you made in package.json

File: src/nobel-prizes/src/meilisearch-client.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const MEILISEARCH_HOST = "https://demos.meilisearch.com"
-const MEILISEARCH_API_KEY = "L4h8HWpdac31f126b8eebbb23e2d10752e86d57d8a388e6910c2fedfb53542547dd4775c"
+const MEILISEARCH_HOST = "https://ms-a79a247a550f-106.lon.meilisearch.io"
```
Comment (bidoubiwa) [inline src/nobel-prizes/src/meilisearch-client.js:1]: Why is it being removed from the demo droplet?

Comment (oluademola) [reply src/nobel-prizes/src/meilisearch-client.js:1]: We are trying to migrate and centralize all the demos to the ms-cloud account Nico created for the demos. We felt that, in the long run, managing the demos in the cloud would be easier due to the upcoming version update capability. For now, this only concerns the demos in this droplet: https://demos.meilisearch.com/. I will share more information about it with you in private. Sorry I haven't informed you about this 😞

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥🔥🔥

**Pull request merged**